README.md

```diff
@@ -1,6 +1,6 @@
 # tarkov.dev 💻
 
-[![deploy](https://github.com/the-hideout/tarkov-dev/actions/workflows/deploy.yml/badge.svg)](https://github.com/the-hideout/tarkov-dev/actions/workflows/deploy.yml) [![ci](https://github.com/the-hideout/tarkov-dev/actions/workflows/ci.yml/badge.svg)](https://github.com/the-hideout/tarkov-dev/actions/workflows/ci.yml) [![CodeQL](https://github.com/the-hideout/tarkov-dev/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/the-hideout/tarkov-dev/actions/workflows/codeql-analysis.yml) ![contributors](https://img.shields.io/github/contributors-anon/the-hideout/tarkov-dev) [![Discord](https://img.shields.io/discord/956236955815907388?color=7388DA&label=Discord)](https://discord.gg/XPAsKGHSzH)
+[![deploy](https://github.com/the-hideout/tarkov-dev/actions/workflows/deploy.yml/badge.svg)](https://github.com/the-hideout/tarkov-dev/actions/workflows/deploy.yml) [![ci](https://github.com/the-hideout/tarkov-dev/actions/workflows/ci.yml/badge.svg)](https://github.com/the-hideout/tarkov-dev/actions/workflows/ci.yml) [![CodeQL](https://github.com/the-hideout/tarkov-dev/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/the-hideout/tarkov-dev/actions/workflows/codeql-analysis.yml) ![contributors](https://img.shields.io/github/contributors-anon/the-hideout/tarkov-dev) [![Discord](https://img.shields.io/discord/956236955815907388?color=7388DA&label=Discord)](https://discord.gg/VGq6xtTD)
```

Comment:
Same thing analytics of users who have joined from this link would be my only understanding why it would be relevant?